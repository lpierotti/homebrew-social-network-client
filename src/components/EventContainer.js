import React from 'react'
import { Segment, Divider } from 'semantic-ui-react'
import { Link } from 'react-router-dom'

const EventContainer = (props) => {

	return (
		<div className={'group-events'}>
			<Segment className={'events'}>
				<h1>Group Events</h1>
				<Divider />
				{props.events ? props.events.map((event, index) => <div key={index}><Link to={`/event/${event.id}`}>{event.name}</Link></div>) : null}
			</Segment>
		</div>
	)
}


export default EventContainer