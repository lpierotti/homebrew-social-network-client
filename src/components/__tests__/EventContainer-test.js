import React from 'react';
import { shallow, mount, render } from 'enzyme';
import EventContainer from '../EventContainer';
import renderer from 'react-test-renderer';