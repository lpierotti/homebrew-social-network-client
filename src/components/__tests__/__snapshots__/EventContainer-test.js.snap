// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventContainer component renders correctly 1`] = `
<div
  style={
    Object {
      "float": "right",
      "minHeight": "400px",
    }
  }
>
  <div
    className="ui segment"
    style={
      Object {
        "minHeight": "400px",
        "minWidth": "600px",
      }
    }
  >
    <h1>
      Group Events
    </h1>
    <div
      className="ui divider"
    />
  </div>
</div>
`;
